src/index.css

```diff
@@ -1,22 +1,78 @@
-:root {
-  --primary: #ddd;
-  --secondary: #e5d4d4;
-}
-
-html,
-body {
+/* Reset styles */
+* {
   margin: 0;
   padding: 0;
-  background-color: var(--primary);
+  box-sizing: border-box;
 }
 
+/* Main container */
 body {
   display: flex;
-  justify-content: center;
-  align-items: center;
-  min-height: 100vh;
+  flex-direction: column;
+  height: 100vh;
 }
 
-* {
-  box-sizing: border-box;
+/* Common padding for header, main, and footer */
+header,
+main,
+footer {
+  padding: 21px;
+}
+
+/* Main content grid */
+main {
+  display: grid;
+  grid-template-columns: 46% 46%;
+  gap: 3%;
+  height: 100%;
+}
+
+/* Styling for sections */
+section {
+  padding: 21px 0;
+}
+
+/* Header styles */
+.show-head {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+/* Styling for scores list */
+.scores-list {
+  width: 100%;
+  border: 1px solid black;
+}
+
+/* Styling for list items */
+li {
+  display: flex;
+  list-style: none;
+}
+
+/* Alternate row background colors */
+.scores-list li:nth-child(odd) {
+  background-color: #f0f0f0;
+}
+
+.scores-list li:nth-child(even) {
+  background-color: #d3d3d3;
+}
+
+/* Input styles */
+.input-name-score input {
+  width: 100%;
```

Comment:
- [ ] I want to kindly suggest that you reduce the length of the input so that it closely matches the provided figma template and also provide the appropriate spacing between inputs as can be seen in the image below
![wfi](https://github.com/mj-e-boc/Leaderboard-setup-project/assets/89055502/cc796dd5-6f7c-4eae-a5ac-c2c721357bcf)